test/integration/istio_http_integration_test.cc

```diff
@@ -184,6 +234,7 @@ class IstioHttpIntegrationTest : public HttpProtocolIntegrationTest {
 
   void SetUp() override {
     config_helper_.addFilter(MakeMixerFilterConfig());
+    config_helper_.addFilter(MakeRbacFilterConfig());
```

Comment:
Will this rbac filter impact other tests?   Is it possible only add this filter specific for your rbac test?

Reply:
It could have impact on other tests but if anything goes wrong, `TEST_P(IstioHttpIntegrationTest, RbacDeny)` should catch that.

Also note this file is already testing all filters together (jwt, authn, rbac and mixer), and I can't test RBAC filter without using authn and jwt filter, so I think it makes more sense to put it here.

Reply:
OK then